examples/iam_policy.json

```diff
@@ -192,14 +192,41 @@
         {
             "Sid": "list",
             "Action": [
-                "s3:ListBucket"
+                "s3:ListBucket",
+                "s3:ListAllMyBuckets",
+                "s3:GetBucketLocation"
             ],
             "Effect": "Allow",
             "Resource": [
-                "arn:aws:s3:::test_bucket_write_only",
+                "arn:aws:s3:::alpha-athena-query-dump",
```

Comment:
@isichei is this supposed to be in the `iam_policy.json` example?

Reply:
Yup